css/civicrm.css

```diff
@@ -19,6 +19,14 @@
   position: absolute;
 }
 
+/* CMS specific padding for sticky header padding */
+.wp-toolbar {
+  --crm-sticky-top: 40px;
+}
+.wp-toolbar .crm-menubar-below-cms-menu {
+  --crm-sticky-top: 72px;
+}
+
```

Comment:
If these are the only classes that define this variable, this will mean `.crm-sticky thead` below won't have a value for other CMSs than WordPress? ie it will only work on WordPress? (which would obvs be fine if this was all in the WP package)

Reply:
@shaneonabike Also don't forget to accommodate responsiveness, i.e. the WordPress admin menu becomes 48px tall below a certain breakpoint.

Reply:
@christianwach right good point. I'll test that and add the proper responsive sizes.

@vingle that's correct. At present this new tag is only used in the Wordpress page. My idea was to roll this out, and then in the other ticket I created add the corresponding sizes for the other systems. I can't really handle Joomla but I can do Drupal (since we have that build here). But I wanted to slowly roll it out to tackle this one problem, and then grow it accordingly... does that sound reasonable?

Short version: Right now this new tag I created isn't being used anywhere except the Wordpress Access table. Once we create a merge request for the rest it could include the new sizes for Drupal/Joomla/you name it

Reply:
@shaneonabike - then I'm afraid I think it's better this goes in the WordPress package's civicrm.admin.css file. 

At the moment there aren't any CMS-specific overrides in civicrm.css (or maybe one - that shouldn't be there) and I'd rather not plan to add a series of them, given there's a plan to to replace civicrm.css.

Do you know what other tables use `.crm-sticky` and `sticky-header`? Is it possible it's just the WordPress Permissions table? I can't picture other instances.

Reply:
@vingle The `civicrm.admin.css` file doesn't load on "standard" CiviCRM admin screens, so it can't go in there.

Reply:
@vingle No, there's no equivalent as things stand. The CiviCRM-WordPress package [adds some admin pages](https://github.com/civicrm/civicrm-wordpress/pull/231) that are specific to WordPress. That CSS file is loaded there and for the Dashlet.

Reply:
Hi folks,

I think that I am going to make everyone happy! I just noticed that there is a JS function that basically determines whether to position the actual CiviCRM menu. In doing so, it drops a ```--crm-menubar-bottom``` inline to basically indicate the bottom of the menu bar. I changed the code to use that since it's 100% responsive as I scale the window. Also it works for Drupal (probably Joomla but I don't have an instance to test).

Then at ```782px``` the menu gets fixed to the top (same in Drupal). So I added a media catch for that and fixed the heights. The Drupal class isn't perfect but should work since there is no class specifically saying there's an admin so I went for whether the menubar is there or not.

@vingle Sticky is being used (the old way) in several areas if you grep that. Mostly, legacy search but also ```Grant, Searches, Financial pages````, but I could see this being slowly applied all around not that it's there even in SearchKit. :fireworks: 

cc @christianwach @vingle